analysis_driver/pipelines/common.py

```diff
@@ -83,19 +83,14 @@ def stage(cls, **params):
 class SampleDataOutput(segmentation.Stage):
     output_fileset = segmentation.Parameter()
 
-    @property
-    def output_cfg(self):
-        output_file_config.set_pipeline_type(self.output_fileset)
-        return output_file_config
```

Comment:
The output file config no longer changes state, so this is not happening at all anymore. Instead of, e.g:
```python
output_file_config.set_pipeline_type('human sample processing')
vcf_stats = output_file_config['vcf_stats']
```
we now do:
```python
vcf_stats = output_file_config['human sample processing']['vcf_stats']
```